src/index.js

```diff
@@ -6,16 +6,28 @@ import App from './Components/App';
 import {
   ApolloProvider,
   ApolloClient,
-  createHttpLink,
+  ApolloLink,
+  concat,
+  HttpLink,
   InMemoryCache
 } from '@apollo/client';
 
-const httpLink = createHttpLink({
+const httpLink = new HttpLink({
   uri: 'https://not-fur-gotten-be.herokuapp.com/graphql'
 });
 
+const operationNameLink = new ApolloLink((operation, forward) => {
```

Comment:
Magic. The other half we needed in order to intercept/stub our data for cypress testing.